src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -47,6 +49,8 @@ class ConsignmentRouteSpec extends AnyFlatSpec with Matchers with TestRequest wi
   val runTestMutation: (String, OAuth2BearerToken) => GraphqlMutationData = runTestRequest[GraphqlMutationData](addConsignmentJsonFilePrefix)
   val expectedQueryResponse: String => GraphqlQueryData = getDataFromFile[GraphqlQueryData](getConsignmentJsonFilePrefix)
   val expectedMutationResponse: String => GraphqlMutationData = getDataFromFile[GraphqlMutationData](addConsignmentJsonFilePrefix)
+  val runParentFolderTestQuery: (String, OAuth2BearerToken) => GraphqlQueryData = runTestRequest[GraphqlQueryData](getParentFolderAndFilesJsonFilesPrefix)
+  val expectedParentQueryResponse: String => GraphqlQueryData = getDataFromFile[GraphqlQueryData](getParentFolderAndFilesJsonFilesPrefix)
```

Comment:
Are these two values used anywhere? Or are they not needed because you've added the test of the parent folder to the existing `getConsignment` test?

Reply:
Ah they're not used anywhere, forgot to remove them! :+1: